monitoring/views/missing_columns_notes.view.lkml

```diff
@@ -18,10 +18,7 @@ view: +missing_columns_notes {
       ;;
     link: {
       label: "{% if value != '' %} view on Bugzilla {% endif %}"
-      url: "
-      {% if value != '' %}
-      {{ bug }}
-      {% endif %}"
+      url: "{{ bug }}"
```

Comment:
Maybe it was the newlines in the value that were the problem?  Does making the conditional a one-liner work?
```suggestion
      url: "{% if value != '' %}{{ bug }}{% endif %}"
```

Reply:
It doesn't work

![image](https://github.com/user-attachments/assets/d0ecd8c3-d790-4489-aa61-df09b07688e5)